department-of-veterans-affairs/caseflow
# Refactored to display virtual hearing fields different for Video hear…

Resolves #14711 

### Description
Places the veteran and POA/Representative email side-by-side for Video hearings, but preserves the sectioning for Central hearings.

### Acceptance Criteria

### Testing Plan
1. Shadow user `BVASYELLOW`
1. Navigate to the hearings schedule and select a Video hearing: http://localhost:3000/hearings/schedule
1. Click the 'Edit Hearing Details' link
1. Change the type to 'Virtual'
1. Ensure the veteran email and POA/Representative email are side-by-side
1. Repeat the steps above choosing instead a Central hearing
1. Ensure that the timezone is displayed for both veteran and POA/Representative and that they are divided into sections

### User Facing Changes

 BEFORE|AFTER
 ---|---
![BeforeVirtualHearingDetails](https://user-images.githubusercontent.com/61989526/88580970-518d9780-d001-11ea-8764-b1b81b3c2ce7.png)|![AfterVirtualHearingDetails](https://user-images.githubusercontent.com/61989526/88580978-54888800-d001-11ea-9153-9c842dd1d927.png)
